pipeline/daily/stages.groovy

```diff
@@ -64,7 +66,17 @@ python host_os.py    \
            --updater-email $params.GITHUB_BOT_EMAIL \
            --push-repo-url $VERSIONS_PUSH_REPO_URL \
            --push-repo-branch $COMMIT_BRANCH \
-"""
+""", returnStatus: true)
+
+    Integer SUCCESS_EXIT_CODE = 0
+    Integer NO_UPDATES_EXIT_CODE = 56
```

Comment:
This is the only place those are used so far. If we start using them elsewhere, it might be a good idea to create a `returncodes.groovy` file.